scripts/install.js

```diff
@@ -26,7 +26,7 @@ var directories = fs.readdirSync(path.join(__dirname, '../packages'));
 
 // This is a helper method which will auto-install all of the dependencies of
 // each module.
-async.eachLimit(directories, 5, function(directory, callback) {
+async.eachSeries(directories, function(directory, callback) {
```

Comment:
This is a bit of backwards progress, but necessary to circumvent an error where each modules install process was trying to write to the same npm cache location. Evidently, there isn't a locking mechanism in place as there is on UNIX-based systems. A possible solution I found was to --force the npm install, but no luck.